Add `comment` and `table_comment` table schema definition

mechnicov opened the pull request:
### Motivation / Background

This Pull Request has been created because these methods are missing, but could be helpful for bulk changes of the table. This will allow you to add comment(s) to the table and/or to the column(s) using single block, improving readability and performance because of single query

### Detail

This Pull Request adds `comment` and `table_comment` for `change_table` definition

```ruby
change_table :users, bulk: true do |t|
  t.comment :login, "Username"
  t.table_comment "Usual users"
end
```

### Additional information

This was intended as non-breaking change

### Checklist

* [x] This Pull Request is related to one change. Unrelated changes should be opened in separate PRs.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [x] Tests are added or updated if you fix a bug or add a feature.
* [x] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

mechnicov committed:
Add `comment` and `table_comment` for `change_table` definition

mechnicov commented:
May be `t.column_comment` (or just `t.comment`) and `t.table_comment` is better

mechnicov commented:
Hello @rafaelfranca

I see you merged similar commit

https://github.com/rails/rails/commit/f84e1ebb5ad79ee53a3a7c080f743ca9cc0fe941

Please review my PR as well. Thank you

zzak commented:
I don't think this is the same as `validate_check_constraint`, and is not supported for all adapters.

mechnicov commented:
Hello @zzak. Thank you for your reply. Currently in the abstract adapter there are [exceptions](https://github.com/rails/rails/blob/965b8c372ac0e19021f92cf798095af6046ab140/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#L1494-L1512) when comments are not supported and specific methods for [PostgreSQL](https://github.com/rails/rails/blob/965b8c372ac0e19021f92cf798095af6046ab140/activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb#L462-L474) and MySQL [1](https://github.com/rails/rails/blob/965b8c372ac0e19021f92cf798095af6046ab140/activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb#L328-L332), [2](https://github.com/rails/rails/blob/965b8c372ac0e19021f92cf798095af6046ab140/activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb#L398-L401) adapters

Maybe you have some idea how to improve PR changes?

mechnicov commented:
I've added condition for tests (only for adapters that support comments). As for the `lib` code, the issue with adapters that don't support comments will be resolved at the schema statement level (as I wrote above, there will be exceptions if the appropriate methods are not defined in the specific adapter). I've also changed naming (`t.comment` rather than `t.change_column_comment` looks like the rest of existing methods)

@zzak could you look at the PR?
